src/bulkActionPopup/scss/bulkActionPopup.scss

```diff
@@ -69,4 +51,14 @@
     .multiple .resource-count{
         font-weight : bold;
     }
+
+    .message{
+        @include flex-container();
+        @include vendor-prefix(align-items, center, property);
```

Comment:
This repo is using cssprefixer, no need to prefix anymore :)

Reply:
fixed